frontstage/controllers/conversation_controller.py

```diff
@@ -72,10 +72,11 @@ def send_message(message_json):
     try:
         response = session.post(url, headers=headers, data=message_json)
         response.raise_for_status()
-    except HTTPError as ex:
+        logger.info("Message sent successfully")
+        return response.json()
+    except HTTPError:
         logger.exception("Message sending failed due to API Error")
-        raise ApiError(ex.response)
-    logger.info("Message sent successfully")
+        raise ApiError(response)
```

Comment:
`raise ApiError(response)` throws the classic `Local variable might be referenced before assignment` warning so might be worth setting response to none prior to the try/except?

Reply:
I think you can put the post outside the try/except to fix the issue.  I think that's what we do in response-operations-ui but you'll have to double check